src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerPreparedStatement.java

```diff
@@ -367,7 +367,6 @@ public void setBigDecimal(int parameterIndex, BigDecimal x, int precision, int s
     public void setLong(int parameterIndex, long x, boolean forceEncrypt) throws SQLServerException;
 
     /**
-     * <p>
```

Comment:
Why are we removing this?

Reply:
why is it necessary? this isn't  in any of the other methods